domains/mwa.json

```
@@ -1,9 +1,10 @@
 {
   "owner": {
     "username": "Totallynotmwa",
-    "email": "totallynot@mwa.lol"
+    "email": "hshdhddzjsjs@gmail.com"
```

Since you don't want to use email you could do this (also you forgot to add a extra comma)
```suggestion
    "email": "",
```